config/styleguide/config.visualdiff.js

```diff
@@ -7,6 +7,7 @@ module.exports = Object.assign({}, config, {
   require: config.require.concat([path.resolve(__dirname, 'assets/disable-animations.css')]),
   propsParser: () => ({}), // Faster without parsing props. They're not needed for diffs.
   theme: {
+    maxWidth: 632,
```

Comment:
What does this key do?

Reply:
It constrains the width of the styleguide, so that our screenshots stay the same width. 
![yamui_v8_5_0](https://user-images.githubusercontent.com/601657/41943062-d6a6d1ba-7956-11e8-8ea1-abf02de4ac39.png)

Reply:
Ah, understood.